### PR TITLE
Change identify to alias so that Mixpanel funnels don't break

### LINKED ISSUE #420: Mixpanel funnels broken
When you sign up for a mailchimp it should call alias rather than identify so the funnel doesn't break for arfah's landing pages

### PR DESCRIPTION
**Pull request by pitchinvasion**
https://mixpanel.com/docs/integration-libraries/using-mixpanel-alias

This will fix landing page funnels

Closes #420

### CODE REVIEW

**Comment by pitchinvasion** [comment]
@spike01 you can just merge this if it looks ok

**Comment by pitchinvasion** [comment]
actually hold off got some more stuff to do

File: source/women.html.haml [hunk 1]
```diff
@@ -84,3 +84,4 @@
 / Mixpanel event for landing page
 :javascript
   mixpanel.track('Viewed Womens Landing Page');
+  mixpanel.people.set({'Category': 'woman'});
```
Comment (spike01) [inline source/women.html.haml]: Does this need to be consistent with the naming elsewhere, i.e. women

Comment (spike01) [inline source/women.html.haml]: Also, not super important, but everything else on Mixpanel is capitalized

**Comment by spike01** [comment]
Just a small thing about consistency otherwise :shipit:

**Comment by tansaku** [comment]
looks okay to me, but don't really feel able to assess ATM ...

**Comment by pitchinvasion** [comment]
Cool all good just need to run by @Arfah I did your changes spike

**Pull request merged**